live-build/misc/ansible-roles/appliance-build.delphix-ldap/tasks/main.yml

```diff
@@ -30,6 +30,13 @@
     - nfs-common
     - nscd
 
+- name: Prevent NSDC from ignoring the CLI NSS module
```

Comment:
I think you meant "NSCD" instead of NSDC. Also, I'm not sure if this should be capitalized; [the man page](http://manpages.ubuntu.com/manpages/bionic/man8/nscd.8.html) uses lowercase letters. I'd personally describe this program with the identifier `nscd(8)` (a Solaris convention, but a generally useful one in my opinion).